http/src/test/resources/reference.conf

```diff
@@ -114,7 +114,8 @@ proxy {
 
 refererConfig {
   validHosts = [
-    "example.com"
+    "example.com",
+    "*:9099"
```

Comment:
Added support for `*:port` in `refererConfig`. 

This is only used in dev/qa, and is needed because the Referer hostname is dynamic based on the Jenkins test run (see above comment).

Reply:
Just have question about the * notation, does this wildcard work for other operations as well? Haven't seen this before so just want to know how it works

Reply:
The way it's implemented `*` only works for the hostname, not the port. It's to handle the case where automation test referers come from different hosts but are always the same port.

Perhaps I can add a comment to this effect in the fc-develop PR.